README.md

```diff
@@ -211,8 +215,8 @@ If Docker is installed, you can build the image with the following command.
 docker build . -t <image-name>
```

Comment:
If you do not want to pull a pre-built image use the docker command bellow to build the Monitoring Exporter image